pkg/winperfcounters/watcher.go

```diff
@@ -151,24 +151,42 @@ func ExpandWildCardPath(counterPath string) ([]string, error) {
 	return win_perf_counters.ExpandWildCardPath(counterPath)
 }
 
-func removeTotalIfMultipleValues(vals []CounterValue) []CounterValue {
+// cleanupScrapedValues handles instance name collisions and standardizes names.
+// It cleans up the list in-place to avoid unnecessary copies.
+func cleanupScrapedValues(vals []CounterValue) []CounterValue {
 	if len(vals) == 0 {
 		return vals
 	}
 
-	if len(vals) == 1 {
-		// if there is only one item & the instance name is "_Total", clear the instance name
-		if vals[0].InstanceName == totalInstanceName {
-			vals[0].InstanceName = ""
-		}
+	// If there is only one "_Total" instance, clear the instance name.
+	if len(vals) == 1 && vals[0].InstanceName == totalInstanceName {
+		vals[0].InstanceName = ""
```

Comment:
I am also curious, could it be related to semconv, or maybe an attempt to avoid a redundant label? 

I don't personally think the extra instance name label is problematic, so I'm not opposed to removing this code, but it might cause issues for existing collectors by creating new time series. 

If we do remove this, it should probably be documented (maybe not as a breaking change, but just so people know to expect their time series to change)